study.md

```diff
@@ -54,5 +58,5 @@ like enumerables?  How are they different?  Please compare and contrast these
 types.
 
 ```md
-<!-- your answer here -->
+Streams are like enumerables because contain data elements that must be worked on one at a time. They are different becuase they are potentially endless sources of data whereas an enumerable will refer to something finite like an array.
```

Comment:
An enumerable will usually refer to something finite, but not always.  An example of an infinite enumerable is one that  returns a digit of π on each iteration.